templates/includes/banner.html

```diff
@@ -0,0 +1,9 @@
+<section class="p-strip--light is-shallow">
+  <div class="row">
+    <p class="u-no-padding u-no-margin u-no-max-width">
```

Comment:
Instead of using `<p>` and removing all styling form it with utilities why not just use `<div class="col-12">`?

Reply:
Oh that's way simpler! I'll update it now